src/ontology/mondo-ingest-odk.yaml

```diff
@@ -33,6 +33,8 @@ components:
       source: https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
     - filename: icd10who.owl
       source: https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl
+    - filename: icd11.owl
```

Comment:
## Linearization question: edges or nodes too?
- [ ] @matentzn 

Just wanna double check my understanding. I just realized that I'd previously thought that the differences between the foundation and linearizations were purely edges; that there are 0 nodes in any linearization that are not in the foundation.

Do you think my understanding is incorrect? If, rather, my understanding is correct, then it seems to me that we do not need to separate namespaces for "icd11" and "icd11 foundation".